Changelog.md

```diff
@@ -1601,7 +1601,7 @@ Language Features:
 
 Compiler Features:
  * ABI: Additional internal type info in the field ``internalType``.
- * eWasm: Highly experimental eWasm output using ``--ewasm`` in the commandline interface or output selection of ``ewasm.wast`` in standard-json.
+ * eWasm: Highly experimental eWasm output using ``--ewasm`` in the commandline interface or output selection of ``ewasm.waste`` in standard-json.
```

Comment:
This fix is wrong, it is not a typo. `.wast` is a file extension for WebAssembly Script Text: https://github.com/WebAssembly/spec/tree/main/interpreter#running-modules-or-scripts.